pkg/quota/resources_test.go

```diff
@@ -266,9 +283,23 @@ func TestIsZero(t *testing.T) {
 			},
 			expected: true,
 		},
-		"non-zero": {
+		"non-zero-both": {
```

Comment:
I don't see the unique branch this tests

Reply:
ok,If the unique branch case no need to test,I will close the pr!thanks!